package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@bpmn-io/extract-process-variables",
+  "version": "0.0.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```
Let's update this before release 😅

Reply:
wouldn't it be updated by np automatically?

Reply:
Could be. We can leave it as it is.